app/src/main/java/com/infomaniak/mail/ui/newMessage/encryption/EncryptionLockButtonView.kt

```diff
@@ -134,7 +108,21 @@ class EncryptionLockButtonView @JvmOverloads constructor(
         binding.unencryptedRecipientText.text = count
     }
 
-    private enum class EncryptionDisplayStyle {
-        ChipIcon, ToolbarButton
+    /**
+     * Add a delay before displaying the loader, to avoid small blink at draft's opening
+     */
+    private fun loadWithDelay() {
+        loadingJob = CoroutineScope(Dispatchers.Default).launch {
+            delay(200)
+            Dispatchers.Main {
+                binding.unencryptedRecipientLoader.isVisible = true
+                binding.unencryptableGroup.isGone = true
+            }
+        }
+    }
```

Comment:
Usually, to achieve that, we use a `Utils.createRefreshTimer()`.
But this way is fine with me too.